libs/dh/admin/feature-user-management/src/lib/tabs/roles-tab/dh-roles-tab.component.scss

```diff
@@ -16,10 +16,21 @@
  */
 :host {
   display: block;
-  // TODO: Add spacing variable for 24px
   margin: 24px var(--watt-space-s);
 }
 
+.user-roles {
+  &__spinner {
```

Comment:
Question: Haven't seen the conversion with __ in class names. Do we use this elsewhere?

Reply:
yeah two other places, it is copy paste

Reply:
For context, this class name is following [BEM](https://css-tricks.com/bem-101/) methodology. We are not really consistent with BEM, but it's being used in quite a few places.